client/src/components/ProductList/ProductList.js

```diff
@@ -1,31 +1,34 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
+import { NavLink } from 'react-router-dom'
 
-export default function ProductListing({
-  img,
-  brand,
-  model,
-  price,
-  sale = false,
-}) {
+export default function ProductList(product) {
```

Comment:
This should be named `props` as it represents React's injected props. You could also directly use destructuring assignment as we did in our course: `ProductList({product}) { ...`